<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stop_view</class>
 <widget class="QWidget" name="stop_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>210</y>
     <width>880</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
	background-color: black; 
	border: 2px solid #ffe200;
}

QTableWidget::item {
    border: 2px solid #ffe200;
    border-radius: 4px;
    color: #ffe200;
}

QTableWidget::item:selected {
	background-color: #ffe200;
	color: black;
}

QHeaderView {
	background-color: black;
}

QHeaderView::section {
	background-color: black;
    border: none;
    height: 32px;
    font-size: 14;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </row>
   <column>
    <property name="text">
     <string>ДАТА</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ПРОДЛЖИТ.</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>КООРД.</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>24.04.22</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>123:00</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>95 -46</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_home">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>460</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Панель навигации"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/res/icons/home.svg</normaloff>:/res/icons/home.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>180</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ПРОСТОИ С НЕОПРЕДЕЛЕННОЙ ПРИЧИНОЙ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>10</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ТЕКУЩИЙ ПРОСТОЙ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ДАТА</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>27.04.22 12:06:21</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ПРОДОЛЖИТ.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>02:56</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>НАЖМИТЕ ДЛЯ ИЗМЕНЕНИЯ ПРИЧИНЫ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_icon_flv">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/res/icons/clip.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>40</y>
     <width>410</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #ffe400;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>НЕОПРЕДЕЛЕННЫЙ ПРОСТОЙ</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
